reference/multipart.md

```diff
@@ -3,33 +3,35 @@ layout: page
 title: Multipart Encoding
 ---
 
-Praxis has built-in support for handling "multipart/form-data" (link to RFC?) encoded requests and responses, in the form of the `Praxis::Types::MultipartArray` type, and the `Praxis::Responses::MultipartOk` response.
-
+Praxis has built-in support for handling "multipart/form-data" (link to RFC?) encoded requests and responses.
```

Comment:
Maybe add "as per [RFC2388](https://www.ietf.org/rfc/rfc2388.txt)" and kill the "link to" question.